package-res/resources/web/pentaho/visual/base/model.js

```diff
@@ -241,6 +180,34 @@ define([
           }
           return result && result.isRejected ? result : ActionResult.fulfill();
         },
+
+        /**
+         * Executes the will/did/rejected event loop associated with a given action.
+         *
+         * @param {function} coreAction - The action to be executed.
+         * @param {pentaho.visual.base.events.Will} will - The "will:" event object.
+         * @param {function} Did - The constructor of the "did:" event.
```

Comment:
`Class.<pentaho.whatever.DidEvent>`
Changing the var to DidEvent would also help readability.